runners/direct-java/src/main/java/org/apache/beam/runners/direct/ModelEnforcement.java

```diff
@@ -28,7 +27,7 @@
  *
  * <p>ModelEnforcement is performed on a per-element and per-bundle basis. The
  * {@link ModelEnforcement} is provided with the input bundle as part of
- * {@link ModelEnforcementFactory#forBundle(CommittedBundle, AppliedPTransform)}, each element
+ * {@link ModelEnforcementFactory#forBundle(DirectRunner.CommittedBundle, AppliedPTransform)} each element
```

Comment:
This break the build because the line is longer than 100 characters. Please fix.